docs/manual/guides/module-listing.de.md

```diff
@@ -320,15 +320,16 @@ Das bisherige Template »list_default_member.html5« ergänzen wir wie folgt:
 <?php else: ?>
 	<div id="MYMEMBERMAP" class="block" style="height:40vh"></div>
 
+	<?php $tmpMemberMapData = '' ?>
 	<?php foreach ($this->tbody as $class => $row): ?>
 		<div class="block memberitem <?= $class ?>"><p>
-		  <a href="mailto:<?= $row['email']['raw'] ?>">
-		  <?= $row['firstname']['content'] ?> <?= $row['lastname']['content'] ?></a>
-		  <span><?= $row['street']['content'] ?> - 
-		  <?= $row['postal']['content'] ?> <?= $row['city']['content'] ?></span>
+		  <a href="mailto:<?= $row['email']['raw'] ?? ''?>">
+		  <?= $row['firstname']['content'] ?? ''?> <?= $row['lastname']['content'] ?></a>
+		  <span><?= $row['street']['content'] ?? '' ?> - 
+		  <?= $row['postal']['content'] ?? '' ?> <?= $row['city']['content'] ?? ''?></span>
```

Comment:
These are not nullable fields in the database, can they ever be null here?

Reply:
If the string in backend is empty the key would not exist. Seems a bug to me.

Reply:
youre are right, my client has forgotten to select `postal` in den module. I get a warning that the array key dosnt exist.